### ISSUE TITLE
How to set a parent folder for a new requirement

### ISSUE THREAD

**Issue by DieterWagner42** [OSLC/oslc4net#20]
Hi,
Thanks all for the good work, but now I want to create a new requirement in a existing folder other than Root Folder.
I don't find any support in the client.
Regards
Dieter

**Comment by berezovskyi** [comment]
Thanks for using OSLC4Net, Dieter! If you solved the problem, I am happy to merge a patch into the lib.

**Comment by berezovskyi** [comment]
@DieterWagner42 just in case you still have the old code, I restarted the maintenance of the library and would be happy to know how/if you solved your problem and what kind of library support you'd need to make the task simpler.

**Comment by DieterWagner42** [comment]
@berezovskyi yes i mangaged it but I need to revisit my code. I also added a lot of interfaces to support global configuration.

**Comment by berezovskyi** [comment]
@DieterWagner42 we've migrated the project to NET 8.0 (though libraries target NETStandard 2.0) and planning a new release. Could you please take another look at the your code and check how much you could open source? GC interfaces would be very appreciated too! Thank you in advance!

**Comment by berezovskyi** [comment]
I somehow missed to answer on the substance: a requirement can link to a parent via `oslc_rm:decomposes` property or by first creating a requirement and then updating the parent to add an extra `oslc_rm:decomposedBy` link.
